src/docs/010_manual/20_install.adoc

```diff
@@ -89,32 +98,37 @@ To get started, you can run `tasks` as a first command:
 ----
 ./dtcw.ps1 tasks --group=doctoolchain
 ----
+
+NOTE: if you're making use of cmd.exe, please replace `.ps1` with `.bat`. Do this for subsequent commands.
+
 --
 
-This command will show you all tasks available through docToolchain, but first it will make sure that docToolchain is installed:
+This 'tasks' command display you all of the tasks available on docToolchain. It also verify that docToolchain is installed by:
+
+* Checking for a configuration file. It will creates a default one if none is available. It prefers a local installation. 
+
+* Making use of docker if it is available and there is no local installation. Please keep in mind that if this is your first time running docToolchain, the images download could take a while.
 
```

Comment:
Lines 106 - 110

This command shows all docToolchain tasks and verifies that docToolchain is installed by doing the following:
- Checking for a configuration file (and create one if none exists). Prefers a local installation.
- Making use of Docker (if available) where there is no local installation.

NOTE: If this is your first time installing docToolchain, the images download might take a while.

**@Bennykillua Ensure that the right statements are in the right parts of the overall section of text.**